tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -485,6 +485,30 @@ module.exports = {
       }
 
       return this.useXpath().expect.element(collaboratorSelector).to.not.be.present
+    },
+    /**
+     * @param key
+     * @param value
+     * @return {Promise<module.exports.commands>}
+     */
+    setCollaboratorsForm: async function (key, value) {
```

Comment:
If this function just accepts one key `expireDate` why even check it. Can we just call it `setExpireDate()`?

Reply:
yes, ok! :+1: